content/topics/git/version-control-and-scrum/_index.md

```diff
@@ -66,8 +66,7 @@ weight: 15
 
 ## General issue board procedure
 
-Every time you move a ticket from one column to another, you are unassigning yourself. A ticket in the done column is reassigned to the person who did the development work on it.
-Comment on the tickets as a form of taking notes & passing on explanations from one person to another.
+Every time you move a ticket from one column to another, you make a PR and assign a team member for the first review on your code. If the code has no bugs or errors then the ticket moves to final code review so it can be merged.If there are any changes requested comments are made on the PR and then the ticket is moves to reveiw changes requested. Tickets that are merged move to the Done column.
```

Comment:
You are softof describing how we do things. But this isn't how scrum is defined. You can say: 
- here is basically how it works in industrty: blah blah
- agile should be adapeted to fit the team: etc
- here are some adaptations we've found useful: foo